### ISSUE TITLE
@horse_js Count running is giving me this error

### ISSUE THREAD

**Issue by mpmckenna8** [workshopper/levelmeup#75]
While trying to run @'horse_js' Count I'm getting this error even while running the given solution.

Using Node v6.2.1

```
Running "@horse_js Count"...

/usr/local/lib/node_modules/levelmeup/problems/horse_js_count/exec-wrap.js:42
  db = wrapped(db, methodUsed)
     ^

TypeError: Assignment to constant variable.
    at LevelUP.<anonymous> (/usr/local/lib/node_modules/levelmeup/problems/horse_js_count/exec-wrap.js:42:6)
    at emitNone (events.js:86:13)
    at LevelUP.emit (events.js:185:7)
    at /usr/local/lib/node_modules/levelmeup/node_modules/levelup/lib/levelup.js:127:12
    at /usr/local/lib/node_modules/levelmeup/node_modules/abstract-leveldown/abstract-leveldown.js:42:7

```

**Comment by martinheidegger** [comment]
This should be fixed with the recent 1.0.0 release. Would you mind to verify?

**Comment by martinheidegger** [comment]
Assuming its fixed.
